src/lib/bootloader/sections.rb

```diff
@@ -48,7 +48,9 @@ def write
 
   private
 
-    def read_default
+    # @return [String, nil] return default boot path as string or nil if not set or something goes wrong
+    # @note shows error popup if calling grub2-editenv failed
```

Comment:
ah, you are right, there was old documention which mention that it raises Cheetah::ExecutionFailed, which is no longer true